targetconfig.json

```diff
@@ -260,6 +260,12 @@
         ]
     },
     "hardwareOptions": [
+        {
+            "name": "Download to Drive",
+            "description": "Download a copy of your game file to your computer",
+            "imageUrl": "/static/hardware/laptop.png",
+            "variant": "hw---stm32f401"
```

Comment:
why is this compiling to stm32f401?

Reply:
Because we don't have a variant that creates a png and this is the variant attached to the button people are most-likely to select if they use download today. 

@riknoll suggested we would eventually want to do the work to make a .png variant.

Reply:
This needs to be thought through. We don't select variant when saving through save icon. Let's wait on this change for next release.